src/storage/mongodb/TenantStorage.ts

```diff
@@ -121,6 +121,8 @@ export default class TenantStorage {
     const uniqueTimerID = Logging.traceStart(tenantID, MODULE_NAME, 'createTenantDB');
     // Create tenant collections
     await global.database.checkAndCreateTenantDatabase(tenantID);
+    // Create initial settings for tenant
+    await global.database.createInitialSettingsForTenant(tenantID);
```

Comment:
I think putting data inside the DB is not targeted to the DB helpers. 
The right place is in TenantService after `await TenantStorage.createTenantDB(filteredRequest.id);`.